bot.py

```diff
@@ -33,7 +33,8 @@
 @bot.event
 async def on_message(message):
     cmd = message.content[1:].split(" ")[0]
-    if message.content.startswith(prefix) and cmd in del_commands:
+    channel = str(message.channel).split(" ")[0]
+    if message.content.startswith(prefix) and cmd in del_commands and channel != "Direct":
```

Comment:
What is this "Direct" thing doing?

Reply:
Because I am setting up the bot for interaction in DMs with users, I needed to have a way to check if the message is a DM or a message in the guild.
In the case of a DM, you get permission error if you try to delete the message.

In DMs, the `message.channel` containts e.g. "Direct message with korni#XXXX". So this was my way to check if the message was in a channel or a direct message.

Reply:
thanks :)

I guess we could consider commenting ^^

Reply:
You mean in the code? I'd be very delighted if we did that! :)
I just put a short comment on the commit.

Reply:
yea, in the code. I head it's good practice :)